org.lflang/src/org/lflang/generator/c/CPreambleGenerator.java

```diff
@@ -39,32 +39,37 @@ public static String generateIncludeStatements(
     ) {
         var tracing = targetConfig.tracing;
         CodeBuilder code = new CodeBuilder();
-        if (cppMode) {
+        if (cppMode || targetConfig.platformOptions.platform == Platform.ARDUINO) {
```

Comment:
Shouldn't we just set `cppMode` accordingly if platform is `Arduino`?

Reply:
Wasn't sure if we should move away from cppmode since we no longer need the CCpp target (C target works since we are not relying on any building or CMake through LFC). I can modify it to default to CppMode for the other things though!

Reply:
Ah, good point. I hadn't thought of that. What "other things" are you referring to though?

Reply:
Disregard, looks like CppMode is otherwise not needed for Arduino.

Reply:
The Or Conditions can be kept as is or I can force it to enter CppMode (which seems like a waste since there's certain places where I need to override CppMode code with custom Arduino Code)

Reply:
OK, fine to leave "as is" then.